org.eclipse.jdt.core.compiler.batch/build.properties

```diff
@@ -16,7 +16,6 @@ source.. = src/
 output.. = bin/
 bin.includes = META-INF/,\
                .,\
-               about.html,\
-               compiler_tool_fragment.properties
+               about.html
 src.includes = about.html
-jars.extra.classpath = lib/javax18api.jar
+jars.extra.classpath = lib/javax18api.jar,platform:/plugin/org.apache.ant/lib/ant.jar
```

Comment:
Why do the compile requires ant?

Reply:
There was/is an ant adapter in jdt core, but now it can be removed from batch plugin - I will try to do so in a moment

Reply:
> There was/is an ant adapter in jdt core, but now it can be removed from batch plugin - I will try to do so in a moment

Scratch that, the thing **is** in batch compiler in the root package, see `JDTCompilerAdapter`

Reply:
Do sou think the Ant part can be removed from the batch compiler and instead be part of jtd.core, or maybe even better an jdt.compiler.ant module? I think having compiler.batch as standalone and technique agnostic as possible would be of great value.